spec/support/test_helpers/teacher_training_public_api_helper.rb

```diff
@@ -24,6 +24,14 @@ def stub_teacher_training_api_providers(recruitment_cycle_year: RecruitmentCycle
     )
   end
 
+  def stub_teacher_training_api_providers_with_multiple_pages(recruitment_cycle_year: RecruitmentCycle.current_year)
+    page_numbers = [1, 2, 3]
+
+    page_numbers.each do |page_number|
```

Comment:
optional: this could probably just be 

[1,2,3].each do |page_number|

What do you think?

Reply:
Refactored and pushed up :)